bolt12/offer-test.json

```diff
@@ -0,0 +1,29 @@
+[
+    {
+	"comment": "Minimal offer for tips",
+        "node key": "48c6e5fcf499f50436f54c3b3edecdb0cb5961ca29d74bea5ab764828f08bf47",
```

Comment:
I get "024ff5317f051c7f6eac0266c5cceaeb6c5775a940fab9854e47bfebf6bc7a0407"?

This seems OK, but node key is completely different?

```
$ ./devtools/bolt12-cli --allow-deprecated-apis=false decode lno1pg9k66twd9kkzmpqw35hq83pqf8l2vtlq5w87m4vqfnvtn82adk9wadfgratnp2wg7l7ha4u0gzqw
description: minimal tip
node_id: 024ff5317f051c7f6eac0266c5cceaeb6c5775a940fab9854e47bfebf6bc7a0407
offer_id: d281dad93b431939cad48471219735db71224fd2b323670c2f0106deea317f39
```